### DISCUSSION TITLE
Beer

### DISCUSSION THREAD

**Discussion by FakeWood** [1101-GS4719-C/Python-Community-GS4719-C#1876] [General]
Ranked as the third most consumed beverage globally, beer holds a significant place in social gatherings, celebrations, and daily life.
Sometimes be called as liquid bread, beer is made from cereal.
While various grains such as wheat, corn, and rice can be used to brew beer, malted barley is the most commonly chosen cereal for brewing beer.
At the first step, brewer will mixed malted barley with hot water, and the carbohydrate in the malted barley will turn into sugar. After one to two hour, brewer will extract the sugar liquid. And boiled it with hops, the small green flower of hop.
These small flowers bring a variety of characteristics to beer.
Hops not only bring bitterness but also play a crucial role in creating diverse scents and tastes in beer .
Furthurmore, hops will release some acid substence while it is boiled with the sugar liquid. And thus can prevent beer from spoiling.
Once the sugar liquid is ready, it will be added with yeast, the magical little creature that can turn sugar into alcohol.
It may takes a week to several months, depending on the type of yeast and the strength of the beer.
And finally, beer is born!
By the brewing temperature. We can divide Beer into two big category. One is Lager and one is Ale.
Lager is brew under relatively lower temperature around 10 degrees Celsius. And have a clean and crispy taste. Most of the beer you buy in 7-11 is Lager.
On the other hand, Ale is brew at higher temperature around 20 degree Celsius, making it have more robust taste and offer a broader range of flavors.
Beer, an ancient beverage,  one of the oldest alcoholic drinks in human history. Its roots trace back thousands of years and has become an integral part of diverse cultures around the world. Cheers!

**Comment by TobywithDino** [comment]
I personally enjoy drinking beer as well. However, instead of drinking alone, I prefer drinking with my friends and we can chat while drinking. I remembered you said that you weren’t a beer enthusiast until you visited Europe and had at least one mug of beer per day. You also mentioned that what truly made he fall in love with beer was the unforgettable flavor of Czech beer. As someone who has never been to Europe, I really want to taste the beer there and feel the difference between Taiwanese beer and European beer with my own tongue.

I think it is like magic that your tongue feels differently after indulging in so much beer in Europe. Initially, You thought Taiwanese beer was too bitter to handle. But after tasted the superior flavors of European beer, you came back to Taiwan and became a beer lover even though it is not European beer.

I also have some similar experience of changing my tongue. I was 
absolutely not a fan of eggplant. Every time I see a plate of eggplant, I 
will avoid to pick them and try to look away. One time my mom cooked 
dinner for us, I was hungry and ate so fast because it was a delicious 
meal. After dinner, however, my mom told me that there was a plate of 
eggplant and I almost ate half of it. My mom cooked so well that I didn’t 
notice that it was eggplant. After that night, I don’t hate having eggplant 
anymore because of the good flavor my mom cooked for me. That was 
the time when I found out my tongue had a magical transformation.
